discovery/kubernetes/endpointslice_test.go

```diff
@@ -629,6 +629,109 @@ func TestEndpointSliceDiscoveryWithServiceUpdate(t *testing.T) {
 	}.Run(t)
 }
 
+func TestEndpointSliceDiscoveryWithUnrelatedServiceUpdate(t *testing.T) {
```

Comment:
I thought we agreed here https://github.com/prometheus/prometheus/pull/16365#issuecomment-2804668786 that such a test isn't really needed/useful, has anything changed in the meantime?

Reply:
Nope. Let me remove this. The PR is updated.